packages/constants/src/constants.ts

```diff
@@ -1,38 +1,122 @@
+// DOM elements
 export const PORTALS_CONTAINER_ID = 'portals-container';
 
+// Links
 export const SUPPORT_PORTAL_URL =
   'https://jira.commercetools.com/servicedesk/customer/portal/1/create/167';
 
-export const __GLOBAL = 'GLOBAL';
-
-// NOTIFICATIONS
+// Notification actions
 export const SHOW_LOADING = 'SHOW_LOADING';
 export const HIDE_LOADING = 'HIDE_LOADING';
 export const HIDE_ALL_PAGE_NOTIFICATIONS = 'HIDE_ALL_PAGE_NOTIFICATIONS';
-export const DOMAINS = {
+
+// Notifications
+export type TAppNotificationDomain = 'global' | 'page' | 'side';
+export const DOMAINS: {
+  [key: string]: TAppNotificationDomain;
```

Comment:
This now fixes the types of the constant.

Before it was implicitly typed as `{ GLOBAL: string, PAGE: string, SIDE: string }`, which is wrong.